pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -165,17 +165,21 @@ func validateSinkThrottleConfig(values *settings.Values, configStr string) error
 	return json.Unmarshal([]byte(configStr), config)
 }
 
-// MinHighWaterMarkCheckpointAdvance specifies the minimum amount of time the
-// changefeed high water mark must advance for it to be eligible for checkpointing.
-var MinHighWaterMarkCheckpointAdvance = settings.RegisterDurationSetting(
+// HighWaterMinUpdateInterval specifies the minimum amount of time that
+// must have elapsed since the last time a changefeed's high water was
+// updated before it is eligible to updated again.
+var HighWaterMinUpdateInterval = settings.RegisterDurationSetting(
```

Comment:
Why not change the var name to be more like the new name?

Reply:
Good catch, I forgot to update it.